lib/active_record/connection_adapters/mysql2rgeo/schema_creation.rb

```diff
@@ -7,36 +7,13 @@ class SchemaCreation < MySQL::SchemaCreation # :nodoc:
         private
 
           def add_column_options!(sql, options)
-            # By default, TIMESTAMP columns are NOT NULL, cannot contain NULL values,
```

Comment:
All of the logic removed in this PR is present in ActiveRecord. Don't want to have it here because it causes things like `options[:as]` to get repeated in the resulting SQL that is produced.